internal/configload/stats.go

```diff
@@ -18,9 +18,12 @@ import ic "github.com/FerretDB/dance/internal/config"
 
 // stats represents the YAML representation of expected stats.
 type stats struct {
-	Fail int `yaml:"fail"`
-	Skip int `yaml:"skip"`
-	Pass int `yaml:"pass"`
+	UnexpectedFail int `yaml:"unexpected_fail"`
+	UnexpectedSkip int `yaml:"unexpected_skip"`
+	UnexpectedPass int `yaml:"unexpected_pass"`
```

Comment:
We are going back and forth there. Those fields should never be present in the YAML configuration file – we don't expect the unexpected. `fail`, `skip`, and `pass` are expected values. Unexpected values should not be in the file.

We split `config` and `configload` into separate packages exactly to have them separate. Why are we trying to blend them together again?

Reply:
My bad for that confusion you are completely right. It's a temporary fix for when we removed our support for regex, as that logic accounted for Go tests that should be skipped. So, now they will be marked as `UnexpectedSkip`. All those other unexpected fields are redundant so I will remove them. I will also add a comment.

Reply:
Let's remove `UnexpectedSkip` and all associated tests. Adding a comment does not change the fact that we expect unexpected things

Reply:
OK cool.

Reply:
I tried to remove all associated tests with unexpected skips but it didn't work as some tests rely on variables and helpers, etc. that are contained within other files.

I think I have a less tedious approach now.